skglm/datafits/single_task.py

```diff
@@ -334,3 +340,14 @@ def full_grad_sparse(
                     grad_j += - X_data[i] * np.sign(tmp) * self.delta
             grad[j] = grad_j / n_samples
         return grad
+
+    def intercept_update_step(self, y, Xw):
+        n_samples = len(y)
+        update = 0.
+        for i in range(n_samples):
+            tmp = y[i] - Xw[i]
+            if abs(tmp) < self.delta:
+                update -= tmp
+            else:
+                update -= np.sign(tmp) * self.delta
+        return update / n_samples
```

Comment:
just to double check the maths: our definition of the Huber function is 1-smooth, so their is no additional stepsize (apart from the n_samples normalization) ?

Reply:
@mathurinm tu as raison, pour le moment alpha ne varie pas dans le path, et ce n'est pas testé d'ailleurs...
J'aurai éventuellement besoin qu'on s'appelle pour fix ça si tu as du temps à m'accorder